### PR TITLE
use tokenize/detokenize

### PR DESCRIPTION
**Pull request by mxyng**
remove server's infill and system prompt which are unused

### CODE REVIEW

**Review by dhiltgen — APPROVED** [review]

File: llm/llm.go [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-// #cgo windows,arm64 LDFLAGS: ${SRCDIR}/build/windows/arm64_static/libllama.a -static -lstdc++
-// #cgo linux,amd64 LDFLAGS: ${SRCDIR}/build/linux/x86_64_static/libllama.a -lstdc++
-// #cgo linux,arm64 LDFLAGS: ${SRCDIR}/build/linux/arm64_static/libllama.a -lstdc++
+// #cgo CPPFLAGS: -Illama.cpp
+// #cgo LDFLAGS: -lstdc++ -lllama
+// #cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/build/darwin/arm64_static/
+// #cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/build/darwin/x86_64_static/
+// #cgo windows LDFLAGS: -static
+// #cgo windows,amd64 LDFLAGS: -L${SRCDIR}/build/windows/amd64_static/
+// #cgo windows,arm64 LDFLAGS: -L${SRCDIR}/build/windows/arm64_static/
```
Comment (dhiltgen) [inline llm/llm.go:9]: My suspicion is the windows build failure in CI is due to dropping -lstdc++ here.

Comment (dhiltgen) [reply llm/llm.go:9]: Actually, no, you have it a few lines up as a general setting, so it's not that... but the windows CI failure looks like it is related to your change.

File: llm/llm.go [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
-// #cgo CFLAGS: -Illama.cpp
-// #cgo darwin,arm64 LDFLAGS: ${SRCDIR}/build/darwin/arm64_static/libllama.a -lstdc++
-// #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/build/darwin/x86_64_static/libllama.a -lstdc++
-// #cgo windows,amd64 LDFLAGS: ${SRCDIR}/build/windows/amd64_static/libllama.a -static -lstdc++
-// #cgo windows,arm64 LDFLAGS: ${SRCDIR}/build/windows/arm64_static/libllama.a -static -lstdc++
-// #cgo linux,amd64 LDFLAGS: ${SRCDIR}/build/linux/x86_64_static/libllama.a -lstdc++
-// #cgo linux,arm64 LDFLAGS: ${SRCDIR}/build/linux/arm64_static/libllama.a -lstdc++
+// #cgo CPPFLAGS: -Illama.cpp
+// #cgo LDFLAGS: -lstdc++ -lllama
```
Comment (dhiltgen) [inline llm/llm.go:4]: try swapping order on these and see if that resolves it.

Comment (mxyng) [reply llm/llm.go:4]: I'm going to roll this back

**Pull request merged**